html-templates/cbl/teacher-dashboard.tpl

```diff
@@ -11,6 +11,7 @@
     {$allContentAreas = Slate\CBL\ContentArea::getAll()}
     <header class="page-header">
         <div class="header-buttons">
+            <button type="button" data-action="view-the-continua">View the Continua</button>
```

Comment:
@jmealo this should be changed to just an <a> tag with an href and the corresponding javascript deleted. I believe class="button" will make the <a> tag look the same as the button